experimental/python/wheel.bzl

```diff
@@ -177,10 +177,9 @@ refer to the package in other packages' dependencies.
         default = "any",
         doc = "Supported platforms. 'any' for pure-Python wheel.",
     ),
-    "python_tag": attr.string(
-        default = "py3",
-        doc = "Supported Python major version. 'py2' or 'py3'",
-        values = ["py2", "py3"],
+    "python_tag": attr.string_list(
```

Comment:
This will break existing users.

How about adding new "python_tags" attribute as list, marking the old one as deprecated and handling the two possibilities in _py_wheel_impl?

Reply:
Sounds good, I will update accordingly

Reply:
@pstradomski Sorry for taking so long to get back to this. I was thinking, since we already have usage of `PY2AND3` in the codebase instead of using a list and adding a new attribute I could add support for that? 

If we have people already using it we can uppercase/lowercase the value and then split accordingly.

Let me know your thoughts and I will update accordingly.